index.qmd

```diff
@@ -5,40 +5,37 @@ subtitle: "How to use VEDA services or deploy your own platform with the VEDA op
 
 ## Welcome {#welcome}
 
-VEDA ([Visualization, Exploration and Data Analysis](https://www.earthdata.nasa.gov/esds/veda)) is a redeployable data platform to support scientific visualization and analysis.
-By combining interactive storytelling with open science principles, VEDA enables researchers to engage new audiences and share their analysis results effectively.
```

Comment:
Why was this dropped?

Reply:
I see what happened. Nothing is dropped. The 2 lines were simply combined into 1. That change and the rest of the format changes in this file were caused by right-clicking the file and choosing "Open With... > Quarto Visual Editor" in VS Code (with the Quarto Extension installed). It seems that doing that with any qmd file will cause the file to be "properly" formatted.

Since I didn't make any content changes, I'm simply going to revert these changes to avoid confusion and keep the PR focused on its purpose.

Reply:
I'd not worry about that, @chuckwondo. The above change is actually no regression in the rendered version of the site, so please just keep the linter-induced changes in.

Reply:
I agree, the changes are find, I just wanted to confirm that we knew where they came from.